pkg/k8/client/client.go

```diff
@@ -11,11 +15,27 @@ import (
 //Get returns a kubernetes client.
 // If namespace is empty, it will use the default namespace configured.
 // If path is empty, it will use the default path configuration
-func Get(namespace, configPath string) (string, *kubernetes.Clientset, *rest.Config, string, error) {
-	log.Debugf("reading kubernetes configuration from %s", configPath)
+func Get(namespace string) (string, *kubernetes.Clientset, *rest.Config, string, error) {
+	home := os.Getenv("HOME")
+	if runtime.GOOS == "windows" {
+		home := os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
+		if home == "" {
+			home = os.Getenv("USERPROFILE")
+		}
+	}
+
+	kubeconfig := path.Join(home, ".kube", "config")
+	kubeconfigEnv := os.Getenv("KUBECONFIG")
+	if len(kubeconfigEnv) > 0 {
+		kubeconfig = kubeconfigEnv
+	}
+
+	if len(kubeconfig) == 0 {
```

Comment:
this case is no possible, right?

Reply:
yep, changing the check to see if the file exists